suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -908,3 +908,18 @@ export const selectInstacelessUnselectedDevices = memoize((state: DeviceRootStat
 
     return deviceUtils.getSortedDevicesWithoutInstances(allDevices, device?.id);
 });
+
+export const selectDeviceInstance = (state: DeviceRootState) => {
+    const device = selectDevice(state);
+
+    return device?.instance ?? null;
+};
+
+export const selectIsUnauthorizedPassphraseDevice = (state: DeviceRootState) => {
+    const isDeviceConnectedAndAuthorized = selectIsDeviceConnectedAndAuthorized(state);
+    const deviceInstance = selectDeviceInstance(state);
+
+    if (!deviceInstance) return false;
+
+    return !isDeviceConnectedAndAuthorized && deviceInstance > 1;
```

Comment:
Just and idea... What about creating also `selectIsPassphraseDevice` selector that would be returning `deviceInstance > 1`?

It would not change the code, but would make it bit more understandable and reusable. But feel free to resolve this.

Reply:
https://github.com/trezor/trezor-suite/pull/12876/commits/a3937fd599788239f40cd82fa5f816f3b1f98bf5